users/reverseproxy.rst

```diff
@@ -68,6 +68,13 @@ Caddy
         write none
         header none
     }
+    
+    caddy2
```

Comment:
That's part of the code block - probably shouldn't be, right?

Reply:
I wanted to follow your advice of combining caddy into 1 subtitle. I took a second look, and it did look a little weird. I separated them into different subtitles